pkg/deb/postinstall

```diff
@@ -2,3 +2,9 @@
 mkdir -p /var/log/serviced
 chmod 1777 /var/log/serviced
 
+for BINARY in "serviced" "serviced-controller" "serviced-storage"; do
+    chgrp serviced /opt/serviced/bin/${BINARY}
+done
+
+chgrp serviced /etc/default/serviced
+chgrp serviced /opt/serviced
```

Comment:
You are right, it makes sense to recursively change group. And permissions are already set in `makefile`.